<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicEventWindow</class>
 <widget class="QDialog" name="BasicEventWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrevious">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenerate">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlanStartTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Planning start time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlanEndTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Planning end time:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTitle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDuration">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Event title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDuration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Event duration (in minutes):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove this event</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="tePlanStartTime">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="tePlanEndTime">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
